### PR TITLE
added round trip option to google flight agent

### PR DESCRIPTION
**Pull request by Jngai**

### CODE REVIEW

**Comment by Jngai** [comment]
@cantino here is an extra option for round trip airfare.

**Comment by Jngai** [comment]
I tried this locally.

File: app/models/agents/google_flights_agent.rb [hunk 1]
```diff
@@ -16,7 +16,9 @@ class GoogleFlightsAgent < Agent
 
       All the default options must exist. For `infantInSeatCount`, `infantInLapCount`, `seniorCount`, and `childCount`, leave them to the default value of `0` if its not necessary.
 
-      Make sure `date` is in this type of date format `YYYY-MO-DAY`.
+      Make sure `date` and `returned_date` is in this type of date format `YYYY-MO-DAY`.
```
Comment (cantino) [inline app/models/agents/google_flights_agent.rb]: Maybe `YYYY-MM-DD`?

File: app/models/agents/google_flights_agent.rb [hunk 2]
```diff
@@ -69,8 +73,10 @@ def default_options
     form_configurable :infantInSeatCount
     form_configurable :infantInLapCount
     form_configurable :seniorCount
+    form_configurable :roundtrip, type: :boolean
+    form_configurable :returned_date, type: :string
```
Comment (cantino) [inline app/models/agents/google_flights_agent.rb]: `return_date`, since it's not in the past tense, right?

**Comment by Jngai** [comment]
@cantino I changed them.

**Comment by Jngai** [comment]
@Pcsl this will be merged in soon.

**Comment by cantino** [comment]
Looks good, thanks!

**Pull request merged**